hips/utils/wcs.py

```diff
@@ -4,16 +4,20 @@
 import numpy as np
 from astropy.coordinates import SkyCoord
 from astropy.wcs import WCS
+from astropy.coordinates import Angle
 from astropy.wcs.utils import pixel_to_skycoord, wcs_to_celestial_frame
 
 __all__ = [
     'WCSGeometry',
 ]
 
-__doctest_skip__ = ['WCSGeometry']
+__doctest_skip__ = [
+    'WCSGeometry',
+    'WCSGeometry.*'
```

Comment:
Well generally, we should probably first try to write docs examples where the docstests pass.
I think you already saw how if some docs example is untested, it will usually break quickly as we change and refactor the code.
So for now, we have to manually go through and review the docs examples from time to time, e.g. before a release.

For now, do as you like here: fix up the docs example so that it works, or skip in whatever way you like (maybe best to make it as narrow as possible, i.e. just list the method with the doctest you can't get to work)?